server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -66,6 +67,19 @@ impl ShipmentTransferProcessor for CreateInboundShipmentProcessor {
         if linked_shipment.is_some() {
             return Ok(None);
         }
+        // 5.
+        let store = StoreRowRepository::new(connection)
```

Comment:
Actual place where transfers aren't created if invoice created datetime is less than store created date